next.config.js

```diff
@@ -105,6 +105,34 @@ module.exports = withBundleAnalyzer(
 			FLOW_PLAYER_ID: process.env.FLOW_PLAYER_ID,
 			GOOGLE_TAG_MANAGER_ID: process.env.GOOGLE_TAG_MANAGER_ID,
 		},
+		async headers() {
+			if (
+				process.env.PROXY_URL.includes('-qas.') ||
+				process.env.PROXY_URL.includes('localhost:3200')
```

Comment:
Optional; I don't think we care about headers in local

Reply:
yeah, it was mostly to be able to see if the header would be there locally. We could remove it, but this way you'll see the same behavior locally as on qas